articles/azure-app-configuration/howto-import-export-data.md

```diff
@@ -455,10 +455,10 @@ From the Azure portal, follow these steps:
 
     | Parameter          | Description                                                                                                                                                                                                                            | Example                   |
     |--------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------|
-    | Prefix             | Optional. This prefix will be trimmed from the imported keys. A key prefix is the beginning part of a key. Prefixes can be used to manage groups of keys in a configuration store. Prefix will be ignored for feature flags.           | *TestApp:*                 |
+    | Prefix             | Optional. This prefix will be trimmed from the exported keys. A key prefix is the beginning part of a key. Prefixes can be used to manage groups of keys in a configuration store. Prefix will be ignored for feature flags.           | *TestApp:*                 |
+    | Export as reference | Optional. Check to export key-values to App Service as App Configuration references. This exports references to the latest revision of key-value(s) in App Configuration. [Learn more](../app-service/app-service-configuration-references.md) |
+    | At a specific time | Optional. Fill out to export key-values from a specific point in time. This is the point in time of the resource you selected in the previous step and the version of key-values that existed in the resource at the specified time will be exported. Format: "YYYY-MM-DDThh:mm:ssZ".                                                    | *07/28/2022 12:00:00 AM*  |
```

Comment:
Can we reorder all items in the same order as how they are shown in the UI?

Reply:
Please there is an updated screenshot in this PR for the portal UI (The change for this is currently still in deployment.) This moves the checkbox above the date filter as the date filter is now hidden when exporting as reference to app service. The items have been ordered to correspond with this latest version. Could I be missing something?

Reply:
It's quite small in GitHub for me to see. Why is the change anyway?

Reply:
Please I have attached the screenshot below:
<img width="571" alt="export-app-service-resized" src="https://user-images.githubusercontent.com/102393878/199887048-7e785d3d-1252-428d-a99a-94997a8f6a0d.png">

Since the "time" filter is of no use when exporting as reference to app service, we hide it when a user checks the box to export as reference. So in the order from top to bottom, the user should select whether to export as reference or not, before they get to the date field.

Reply:
Thanks for the explanation, @albertofori. The UI becomes a bit cluttered after the change. Let's chat offline to see if there are other options.